app/views/posts/index.html.erb

```diff
@@ -25,7 +25,7 @@
             <% post.comments[1..5].each_with_index do |comment, index| %>
               <div class="user_post_comment">
                 <h3><%= index + 1 %>. <%= comment.user.Name %></h3>
```

Comment:
- [ ] Please fix the attribute name error here as well as the user model is defined with the attribute `name` and not `Name`